### ISSUE TITLE
Unable to read Info.plist and Apple Requires Properties that are Unused

### ISSUE THREAD

**Issue by allidoisace** [dpa99c/cordova-diagnostic-plugin#462]
# Bug report

**CHECKLIST**
- [x] I have read the [issue reporting guidelines](https://github.com/dpa99c/cordova-diagnostic-plugin#reporting-issues)

- [x] I confirm this is a suspected bug or issue that will affect other users

- [ ] I have reproduced the issue using the [example project](https://github.com/dpa99c/cordova-diagnostic-plugin-example) or provided the necessary information to reproduce the issue.

- [x] I have read [the documentation](https://github.com/dpa99c/cordova-diagnostic-plugin) thoroughly and it does not help solve my issue.

- [x] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**
When running ios with ionic I receive this issue after installing the Diagnostic plugin. Now, this is an accurate `fatal` because I don't have an `Info.plist`, but rather I have `development.plist` and `production.plist`. I am only using the `Location` methods out of this plugin and have added the necessary properties to the property lists. In addition, Apple is complaining about not having usage descriptions for Contacts, Calendar, etc, when I do not even use those particular methods.
```
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
```

After installing the Diagnostic plugin to my ionic application I receive the fatal messages above when running the app or building it.

 Neither

**Expected behavior:**

I would expect any `.plist` to be read and also not require descriptions for methods that are not used (Contacts, Calendar, etc.).

**Steps to reproduce:**

Change to different plists and install the Dialog plugin.

**Screenshots**

**Environment information**

- Cordova CLI version 
	- `cordova -v`
- Cordova platform version
	- `cordova platform ls`
- Plugins & versions installed in project (including this plugin)
    - `cordova plugin ls`
- Dev machine OS and version, e.g.
    - OSX
        ```
        ProductName:    macOS
        ProductVersion: 12.2.1
        BuildVersion:   21D62
        ``` 
    - Windows 10
        - `winver`
        
_Runtime issue_
- Device details
    - iPad 12.9in. Pro
- OS details
    - iOS 15.3.1
	
_Android build issue:_	
- Node JS version
    - `node -v`
- Gradle version
	- `ls platforms/android/.gradle`
- Target Android SDK version
	- `android:targetSdkVersion` in `AndroidManifest.xml`
- Android SDK details
	- `sdkmanager --list | sed -e '/Available Packages/q'`
	
_iOS build issue:_
- Node JS version
    - v16.9.1
- XCode version
    - Version 13.2.1 (13C100)

**Related code:**
```
<key>NSLocationAlwaysUsageDescription</key>
<string>Always allow Geolocation?</string>
<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
<string>Always allow Geolocation?</string>
<key>NSLocationWhenInUseUsageDescription</key>
<string>Allow Geolocation?</string>


if (await Diagnostic.getLocationAuthorizationStatus() === Diagnostic.permissionStatus.DENIED_ALWAYS) {
       ...
}
```

**Console output**
<details>
<summary>console output</summary>

```
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
[fatal] Unable to read: /.../app/ios/App/App/Info.plist
```

</details><br/><br/>

**Other information:**

**Comment by dpa99c** [comment]
Please see the [Specifying modules](https://github.com/dpa99c/cordova-diagnostic-plugin#specifying-modules) section of the documentation. 
Before reporting bugs, please validate the plugin behaviour by building and running the [example project](https://github.com/dpa99c/cordova-diagnostic-plugin-example) which is a known code base.
